sympy/parsing/autolev/test-examples/ruletest8.py

```diff
@@ -1,45 +1,45 @@
-point_o = me.Point('o')
-m1 = sm.symbols('m1')
+frame_a = _me.ReferenceFrame('a')
+c1, c2, c3 = _sm.symbols('c1 c2 c3', real=True)
+a=_me.inertia(frame_a, 1, 1, 1)
+particle_p1 = _me.Particle('p1', _me.Point('p1_pt'), _sm.Symbol('m'))
+particle_p2 = _me.Particle('p2', _me.Point('p2_pt'), _sm.Symbol('m'))
+body_r_cm = _me.Point('r_cm')
+body_r_f = _me.ReferenceFrame('r_f')
+body_r = _me.RigidBody('r', body_r_cm, body_r_f, _sm.symbols('m'), (_me.outer(body_r_f.x,body_r_f.x),body_r_cm))
+frame_a.orient(body_r_f, 'DCM', _sm.Matrix([1,1,1,1,1,0,0,0,1]).reshape(3, 3))
+point_o = _me.Point('o')
+m1 = _sm.symbols('m1')
 particle_p1.mass = m1
-m2 = sm.symbols('m2')
+m2 = _sm.symbols('m2')
 particle_p2.mass = m2
-mr = sm.symbols('mr')
+mr = _sm.symbols('mr')
 body_r.mass = mr
-i1 = sm.symbols('i1')
-i2 = sm.symbols('i2')
-i3 = sm.symbols('i3')
-body_r.inertia = (me.inertia(body_r_f, i1, i2, i3, 0, 0, 0), body_r_cm)
+i1 = _sm.symbols('i1')
+i2 = _sm.symbols('i2')
+i3 = _sm.symbols('i3')
+body_r.inertia = (_me.inertia(body_r_f, i1, i2, i3, 0, 0, 0), body_r_cm)
 point_o.set_pos(particle_p1.point, c1*frame_a.x)
 point_o.set_pos(particle_p2.point, c2*frame_a.y)
 point_o.set_pos(body_r_cm, c3*frame_a.z)
-a=me.inertia_of_point_mass(particle_p1.mass, particle_p1.point.pos_from(point_o), frame_a)
-a=me.inertia_of_point_mass(particle_p2.mass, particle_p2.point.pos_from(point_o), frame_a)
-a=body_r.inertia[0] + me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
-a=me.inertia_of_point_mass(particle_p1.mass, particle_p1.point.pos_from(point_o), frame_a) + me.inertia_of_point_mass(particle_p2.mass, particle_p2.point.pos_from(point_o), frame_a) + body_r.inertia[0] + me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
-a=me.inertia_of_point_mass(particle_p1.mass, particle_p1.point.pos_from(point_o), frame_a) + body_r.inertia[0] + me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
-a=body_r.inertia[0] + me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
+a=_me.inertia_of_point_mass(particle_p1.mass, particle_p1.point.pos_from(point_o), frame_a)
+a=_me.inertia_of_point_mass(particle_p2.mass, particle_p2.point.pos_from(point_o), frame_a)
+a=body_r.inertia[0] + _me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
+a=_me.inertia_of_point_mass(particle_p1.mass, particle_p1.point.pos_from(point_o), frame_a) + _me.inertia_of_point_mass(particle_p2.mass, particle_p2.point.pos_from(point_o), frame_a) + body_r.inertia[0] + _me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
+a=_me.inertia_of_point_mass(particle_p1.mass, particle_p1.point.pos_from(point_o), frame_a) + body_r.inertia[0] + _me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
+a=body_r.inertia[0] + _me.inertia_of_point_mass(body_r.mass, body_r.masscenter.pos_from(point_o), frame_a)
```

Comment:
There should be space around the equals that is used to define a variable, see PEP8.

Reply:
Hello @NikhilPappu I resolved all other formatting issues in `_listener_autolev_antlr` but I am struck while resolving the space issues around variables in `ruletest8.py` and `ruletest9.py` test-examples, am not able to locate the corresponding code which I need to alter in `_listener_autolev_antlr` for these spaces.If you could give me any hint for any variable hopefully I will be able to fix other.Also,we didn't altered `_listener_autolev_antlr` which could have raised such issues ,then why are we facing these formatting issues while updating test examples ?

Reply:
@NikhilPappu I hope you're doing well.I am waiting for your guidance.

Reply:
@ayushmankoul You will need to look at all the strings and write statements in `_listener_autolev_antlr` and put spaces around things like `+` signs and stuff (see PEP8).
You can check how this looks by rewriting the test files and going over them.
I will be quite busy till this month end due to exams so I apologize if I am unable to get back to you immediately.

Reply:
@ayushmankoul You do not need to fix formatting issues in `_listener_autolev_antlr`. Keep in mind that `_listener_autolev_antlr` writes out parser code and it is the one that generates all the `ruletest` files. So you would need to modify the code that `_listener_autolev_antlr` outputs and not the file itself. Also, do not modify the `ruletest` files by themselves as it has no effect on the parser.

Reply:
Thank You @NikhilPappu for responding despite being busy with your exams.I am following your advice to alter the parser code but I am struck while debugging the `_listener_autolev_antlr` to put spaces around variable declarations.I tried to crosscheck almost every string and write statements which could have caused this issue but there is already proper space given in `" = "` .  Please if you can have a look on `_listener_autolev_antlr` to figure out exactly what needs to be altered to overcome this issue.

Reply:
@moorepants Please help me, I am struck with this issue here.